huxley/www/static/js/huxley/components/AdvisorRosterView.js

```diff
@@ -0,0 +1,106 @@
+/**
+* Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
+* Use of this source code is governed by a BSD License (see LICENSE).
+*
+* @jsx React.DOM
++*/
+
+'use strict';
+
+var $ = require('jquery');
+var React = require('react');
+var Router = require('react-router');
+
+var AssignmentStore = require('../stores/AssignmentStore');
+var Button = require('./Button');
+var CommitteeStore = require('../stores/CommitteeStore');
+var CountryStore = require('../stores/CountryStore');
+var CurrentUserStore = require('../stores/CurrentUserStore');
+var DelegateStore = require('../stores/DelegateStore');
+var CurrentUserActions = require('../actions/CurrentUserActions');
+var InnerView = require('./InnerView');
+
+var AdvisorRosterView = React.createClass({
+  mixins: [
+    Router.Navigation,
+  ],
+
+  getInitialState: function() {
+    return {
+      assignments: [],
+      delegates: [],
+      committees: {},
+      countries: {},
+      loading: false
+    };
+  },
+
+  componentWillMount: function() {
+    var user = CurrentUserStore.getCurrentUser();
+    AssignmentStore.getAssignments(user.school.id, function(assignments) {
+      this.setState({assignments: assignments.filter(
+        function(assignment) {
+          return !assignment.rejected
+        }
+      )});
+    }.bind(this));
+    CommitteeStore.getCommittees(function(committees) {
```

Comment:
Is there a reason to make an ajax call to get our committees when this page is loaded?